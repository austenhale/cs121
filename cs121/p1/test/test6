Big Jet Plane
Julia Stone
3:54
sounds/westernBeat.wav
Big Jet Plane
Julia Stone
3:54
sounds/westernBeat.wav
Big Jet Plane
Julia Stone
3:54
sounds/westernBeat.wav
