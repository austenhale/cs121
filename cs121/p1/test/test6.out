Enter title: Enter artist: Enter play time (mm:ss): Enter file name: Enter title: Enter artist: Enter play time (mm:ss): Enter file name: Enter title: Enter artist: Enter play time (mm:ss): Enter file name: 

Average play time for songs: 234.00

Song with play time closest to 240 secs: Big Jet Plane

==============================================================================
Title                Artist               File Name                  Play Time
==============================================================================
Big Jet Plane        Julia Stone          sounds/westernBeat.wav           234
Big Jet Plane        Julia Stone          sounds/westernBeat.wav           234
Big Jet Plane        Julia Stone          sounds/westernBeat.wav           234
==============================================================================
